src/DynamoCore/Search/SearchElements/NodeSearchElement.cs

```diff
@@ -18,6 +18,7 @@ public abstract class NodeSearchElement : ISearchEntry, ISource<NodeModel>
         protected string iconName;
 
         private readonly HashSet<string> keywords = new HashSet<string>();
+        protected readonly List<double> keywordsWeights = new List<double>();
```

Comment:
`keywordWeights` - In English, the adjective should not be plural, as that would be redundant. :)